src/content/advanced/cost-optimization/workload-clusters-scale-down/index.md

```diff
@@ -0,0 +1,199 @@
+          containers:
+          - name: silences-customer-create-{CLUSTER-ID}
+            image: quay.io/giantswarm/k8s-initiator
+            imagePullPolicy: IfNotPresent
+            command:
+            - /bin/sh
+            - -c
+            - |
+              cat <<EOF | kubectl apply -f -
+              apiVersion: monitoring.giantswarm.io/v1alpha1
+              kind: Silence
+              metadata:
+                name: {INSTALLATION}-{CLUSTER-ID}-silence-cluster
+                annotations:
+                  'giantswarm.io/keep': 'true'
+              spec:
+                targetTags:
+                - name: installation
+                  value: {YOUR_INSTALLATION_NAME}
+                matchers:
+                - name: cluster_id
+                  value: {CLUSTER_ID}
+                  isRegex: false
+              EOF
+
+              # patch annotations of min and max values for autoscaling to 0
+              kubectl patch MachinePool -n {YOUR_ORGANIZATION} {CLUSTER_ID} --type merge -p '{"metadata" : {"annotations": {"cluster.k8s.io/cluster-api-autoscaler-node-group-min-size": "0"}}}'
+              kubectl patch MachinePool -n {YOUR_ORGANIZATION} {CLUSTER_ID} --type merge -p '{"metadata" : {"annotations": {"cluster.k8s.io/cluster-api-autoscaler-node-group-max-size": "0"}}}'
+
+          restartPolicy: Never
+```
+
+This CronJob will run accordingly to the specified schedule e.g. every Friday att 16:47 UTC. It will first create a silence for the whole cluster, such that the GS oncall is not paged by any incidents happening on the clusters.
+After the Silence creation, the annotations for mininum and maximum scaling limits for autoscaler will be adjusted to 0 what will result in 0 worker nodes in the Node Pool. The patch command in the CronJob can be repeated for multiple Node Pools to obtain 0 workers Wokload Clusters.
+
+### Scale up and delete silence for the cluster
+
+When the time is right to scale up the cluster again, following [template](cronjob-scale-up.yaml) can be applied:
+
+```yaml
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: silences-customer-delete-{CLUSTER-ID}
+  namespace: {YOUR_ORGANIZATION}
+  labels:
+    app: silence-customer
+spec:
+  schedule: "00 8 * * 1" #Timezone - UTC by default - Mondays, 8 AM UTC
+  successfulJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: silences-customer
+        spec:
+          serviceAccountName: silences-customer-rbac
+          containers:
+          - name: silences-customer-delete-{CLUSTER-ID}
+            image: quay.io/giantswarm/k8s-initiator
+            imagePullPolicy: IfNotPresent
+            command:
+            - /bin/sh
+            - -c
+            - |
+
+              kubectl patch MachinePool -n {YOUR_ORGANIZATION} {CLUSTER-ID} --type merge -p '{"metadata" : {"annotations": {"cluster.k8s.io/cluster-api-autoscaler-node-group-min-size": "1"}}}'
```

Comment:
Maybe I would use a more reasonable min value such as 3

Reply:
this is just as example - can be set by customer to whatever they have currently